spec/lib/dfid/govuk_country_registry_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+require 'dfid-transition/govuk_country_registry'
+
+describe DfidTransition::GovUkCountryRegister do
+
+  describe '.countries' do
+    let(:query_results_p1)  { 'spec/fixtures/service-results/country-register-p1.json' }
+    let(:query_results_p2)  { 'spec/fixtures/service-results/country-register-p2.json' }
+
+    def page(index)
+      { 'page-index' => index, 'page-size' => 100}
+    end
+
+    it 'returns active countries from the register' do
+      allow(RestClient).to receive(:get).with(
+        DfidTransition::GovUkCountryRegister::URL,
+        params: page(1)).and_return(File.read(query_results_p1))
+      allow(RestClient).to receive(:get).with(
+        DfidTransition::GovUkCountryRegister::URL,
+        params: page(2)).and_return(File.read(query_results_p2))
+
+      result = DfidTransition::GovUkCountryRegister.countries
+
+      expect(result.count).to eq 195
+      expect(result).to include({
```

Comment:
Perhaps we could reveal some intent by making it obvious that this is an active country...